README.md

```diff
@@ -1,24 +1,158 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Catalog Of My Things
```

Comment:
- [ ] On your readme, please, kindly add a link to the frontend part of the project.
- [ ] Kindly add a live demo section with a link to your deployment.

That way, your project will be well documented.